content/influxdb/cloud/visualize-data/annotations.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Use annotations in dashboards
+description: >
+  Add annotations to your dashboards to provide useful, contextual information about single points in time.
+influxdb/cloud/tags: [labels]
+menu:
+  influxdb_cloud:
+    name: Use annotations
+    parent: Visualize data
+weight: 104
+---
+
```

Comment:
Embed YouTube Tutorial:
<iframe width="560" height="315" src="https://www.youtube.com/embed/5NEplCesNAc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>